### PR TITLE
log slow tests

### PR DESCRIPTION
**Pull request by zeripath**
Add some logging to the integration tests when there is a slow test or slow flush.

Signed-off-by: Andrew Thornton <art27@cantab.net>

### CODE REVIEW

File: integrations/testlogger.go [hunk 1]
```diff
@@ -13,12 +13,17 @@ import (
...
+	"time"
 
 	"code.gitea.io/gitea/modules/log"
 	"code.gitea.io/gitea/modules/queue"
 )
 
-var prefix string
+var (
+	prefix    string
+	slowTest  time.Duration = 10 * time.Second
```
Comment (6543) [inline integrations/testlogger.go]:
```suggestion
	slowTest  = 10 * time.Second
```

File: integrations/testlogger.go [hunk 2]
```diff
@@ -13,12 +13,17 @@ import (
...
+	slowFlush time.Duration = 5 * time.Second
```
Comment (6543) [inline integrations/testlogger.go]:
```suggestion
	slowFlush = 5 * time.Second
```

**Review by 6543 — COMMENTED** [review]
this may fix the lint - but didn't tested it jet

**Review by 6543 — APPROVED** [review]
I like the new log stile

**Review by techknowlogick — APPROVED** [review]

**Comment by techknowlogick** [comment]
Ping lg-tm

**Pull request merged**